blockchain/indexers/common.go

```diff
@@ -142,8 +142,9 @@ type Indexer interface {
 	// WaitForSync subscribes clients for the next index sync update.
 	WaitForSync() chan bool
 
-	// Subscribers returns all client channels waiting for the next index update.
-	Subscribers() map[chan bool]struct{}
+	// UpdateSyncSubscribers closes all sync subscriber update channels
+	// and purges the sync subscriber set.
```

Comment:
I'd suggest framing the interface docs in term of functionality and purpose rather than implementation.